chapters/102/index.rst

```diff
@@ -0,0 +1,174 @@
+================================================
+Jina 102: How basic components work together
+================================================
+
+
+In Jina 102, you’ll learn how Jina’s basic components work together. Before you start, make sure you’ve already read through `Jina 101`_.
+
+.. _Jina 101: https://101.jina.ai 
+
+
+.. contents:: Table of Contents
+    :depth: 3
+
+
+Search workflows
+==================
+
+
+First, we’ll show you how a Flow works in Jina. Be aware, there are many different ways to build a Flow, and this is just one way.
+
+Let’s use a simple text-to-text search application as an example — searching for sentences in a book. We use two Flows for this:
+
+*   An **indexing** Flow makes the whole book searchable by sentence
+*   A **querying** Flow handles a user query (in this case, a sentence) and returns search results
+
+A working example can be found `here`_. (Bear in mind that this example uses pre-processed sentences, so the segmenting step is skipped)
+
+The indexing Flow prepares data to be searched. Input Documents are fed in, processed, and output at the other end as searchable indexes in storage:
+
+.. _here: https://github.com/jina-ai/examples/tree/master/wikipedia-sentences
+
+
+.. image:: /chapters/102/images/index_flow.png 
+
+
+Then, the querying Flow takes a user query as its input Document, and returns a list of ranked matches:
+
+.. image:: /chapters/102/images/query_flow.png 
+
+
+Inside a Flow
+-----------------
+
+Here, we’ll use the indexing Flow as an example:
+
+.. image:: /chapters/102/images/flow_pods.png 
+
+In our Flow, different Pods perform different tasks. Documents (in this case a book) are:
+
+1. Input Documents segmented into chunks (sentences)
+2. Chunks transformed into vector embeddings
+3. Chunks indexed in key-value pairs and saved to storage
+
+Inside a Pod
+----------------------
+
+.. image:: /chapters/102/images/pod_peas.png 
+
+In the Segmenter Pod, there’s only one Pea. But sometimes you group multiple copies of the same Peas in a Pod to increase reliability or performance. To implement sharing and parallelization, check `this section`_.
```

Comment:
one comment is that we should link terms such as `Pod`, `Flow` to our glossary, checkout [here](https://docs.jina.ai/master/chapters/glossary/glossary.html) to see Jina glossaries, and [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html) to find how how to do it.

Reply:
@bwanglzu Since Flow Pod & Pea appear too many times in 102, I would only link them for the first time they appear.

Reply:
sounds good to me! it's a minor thing so I didn't "request change". Feel free to merge.

Reply:
@bwanglzu I added terms in a few places. Feel free to double check.